book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -1,31 +1,362 @@
-Inclusion is more than a buzzword - it’s a commitment to ensuring that every individual regardless of their background, abilities, or current circumstances has equal opportunity to participate and thrive. 
+Inclusion is more than a buzzword - it’s a commitment to ensuring that every individual regardless of their background, abilities, or current circumstances has equal 
+opportunity to participate and thrive. 
 
-For many projects, the involvement of individuals and communities with first-hand lived experience is critical to upholding and ensuring that the processes and outcomes developed are effective, inclusive, and equitable. 
-This can involve hosting different events that are attended by people with lived experience and other members of the public, as well as researchers, clinical staff, and other professionals. 
+For many projects, the involvement of individuals and communities with first-hand lived experience is critical to upholding and ensuring that the processes and outcomes 
+developed are effective, inclusive, and equitable. 
+This can involve hosting different events that are attended by people with lived experience and other members of the public, as well as researchers, clinical staff, 
+and other professionals. 
 
 This section provides guidance and considerations for how to design and host an inclusive event for a diverse audience of people with lived experience. 
 
-The recommendations are not an exhaustive list, and it is always preferable to include a specific representative or person with lived experience in the event planning to ensure that your event is tailored to the needs of your specific attendees. 
+The recommendations are not an exhaustive list, and it is always preferable to include a specific representative or person with lived experience in the event planning 
+to ensure that your event is tailored to the needs of your specific attendees. 
 Their inclusion in decision-making processes is essential to achieving an accessible and inclusive event.
 
-
+(cl-inclusive-events-planning-intro)=
 ## Accessibility vs. Inclusion: What to focus on
 
-When designing and organising an event, it can be easy to focus on the accessibility requirements of the venue and overlook the steps needed to make an event inclusive and enjoyable for all participants. 
+When designing and organising an event, it can be easy to focus on the accessibility requirements of the venue and overlook the steps needed to make an event inclusive 
+and enjoyable for all participants. 
 
-We can think of this in terms of “Universal Design” which is about designing an event or environment so that it can be accessed, understood, and used by all people regardless of their background, age, size, ability or disability. 
+We can think of this in terms of “Universal Design” which is about designing an event or environment so that it can be accessed, understood, and used by all people 
+regardless of their background, age, size, ability or disability. 
```

Comment:
Updated (and also used exactly that example)

I added in links to the universal design website and tried to weave it in so that folks don't feel pressured to read all about universal design in order to host events

Reply:
That's great!